crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -203,11 +188,9 @@ impl UncompressedDataManager {
     }
 
     /// Insert `ingested_data_buffer` into in-memory buffers managed by the storage engine. Returns
-    /// [`String`] if the channel or the metadata Delta Lake could not be read from.
-    async fn insert_data_points(
-        &self,
-        ingested_data_buffer: IngestedDataBuffer,
-    ) -> Result<(), String> {
+    /// [`crate::error::ModelarDbServerError`] if the channel or the metadata Delta Lake could not
```

Comment:
Maybe "[\`ModelarDbServerError\`]\(crate::error::ModelarDbServerError\)" instead. If it is changed here, also change it elsewhere in the file.

Reply:
Fixed.